UnitystationLauncher/ContentScanning/ScanConfigMode.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using ILVerify;
+
+namespace UnitystationLauncher.ContentScanning;
+
+public sealed class SandboxConfig
+{
+    public string? SystemAssemblyName { get; set; }
+    public List<VerifierError> AllowedVerifierErrors { get; set; } = new List<VerifierError>();
+    public List<string> WhitelistedNamespaces { get; set; } = new List<string>();
+    public List<string> MultiAssemblyOtherReferences { get; set; } = new List<string>();
+
+    public Dictionary<string, Dictionary<string, TypeConfig>> Types { get; set; } =
+        new Dictionary<string, Dictionary<string, TypeConfig>>();
+}
+
+public enum InheritMode : byte
```

Comment:
Models/Enums/InheritMode.cs

Reply:
as the class hierarchy so Models, Enums , InheritMode?